cmd/zoekt-sourcegraph-indexserver/index_test.go

```diff
@@ -120,9 +120,10 @@ func TestGetIndexOptions(t *testing.T) {
 
 func TestIndex(t *testing.T) {
 	cases := []struct {
-		name string
-		args indexArgs
-		want []string
+		name                      string
+		args                      indexArgs
+		mockGetRepositoryMetadata func(args *indexArgs) (*zoekt.Repository, error)
```

Comment:
we aren't testing error cases, so you can simplify the specification to just specify a pointer to zoekt.Repository. If non-nil it is used. The more succinct a test case, the easier it is to read the test cases in the future.

style: I don't think we need such a verbose name, I'd prefer something just like `current *zoekt.Repository` or even `current []zoekt.RepositoryBranch`